packages/code-studio/src/styleguide/MockIrisGridTreeModel.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint class-methods-use-this: "off" */
 /* eslint no-unused-vars: "off" */
 /* eslint no-empty-function: "off" */
+// eslint-disable-next-line max-classes-per-file
```

Comment:
Why was this added when nothing else in this file changed?

Reply:
Whoops